### ISSUE TITLE
List of features and wireframes

### ISSUE THREAD

**Issue by tas12** [elischutze/grrrrl#7]
Hi Eli, I really like your idea and love that you've invited others to collaborate. Do you think there should be an issue to suggest features? After that, some wireframes could be submitted?

**Comment by elischutze** [comment]
sounds good to me!

**Comment by tas12** [comment]
Okay! So going by the README, the idea is to have two pages:

Search page:
- search box
- search results

Discover page:
- tech talks by topic (users select a topic and are presented with related talks)

Suggestion (contributors, feel free to suggest otherwise!):
Have 4 pages:
Home page/Landing page:
- Search box
- Short description of what Grrrrl is

Discover page:
- tech talks by topic (this could be designed in a fun and engaging way. I don't have amazing design skills though 😂 )

Results page:
- Display of results based on search string or selected topic

Submit a talk page:
- A form to submit a talk. Input fields could include: Talk name, URL of talk, description, topic

I can submit a pull request to add initial wireframes

**Comment by elischutze** [comment]
Yeah, that sounds great. Definitely important to have a submission page!

**Comment by tas12** [comment]
Ah, just realised some of the other issues already mentioned submission page. @elischutze please could you add me as a contributor so that I can submit a pr?
